src/Uno.UI.Composition/Composition/Visual.skia.cs

```diff
@@ -22,6 +22,37 @@ public partial class Visual : global::Microsoft.UI.Composition.CompositionObject
 	private bool _matrixDirty = true;
 	private Matrix4x4 _totalMatrix = Matrix4x4.Identity;
 
+	// a visual is a flyout visual if it's directly set by SetAsFlyoutVisual or is a child of a flyout visual
+	// This doesn't yet handle the case of a child visual changing parents. i.e., once a flyout visual, always a flyout visual.
```

Comment:
note to self: this comment should be removed since it's already fixed with `_isFlyoutVisualInherited`